machines/run.html.md

```diff
@@ -0,0 +1,368 @@
+
+Here's what `fly machine run` does for you:
+* Checks with the platform for the org and app you've specified, if any, and if needed, guides you through naming a new app
+* Creates a Fly App, if applicable
+* Gets, or builds, a Docker image to make the Machine from
+* Creates the Machine with some default config, plus config you pass to it with flags
+* Starts the Machine
+* Waits for the Machine to start before declaring success (or failure)
+
+To create a Machine, but not start it, use [`fly machine create`](/docs/flyctl/machine-create/).
+
+To make changes to a Machine once it's created or run, use [`fly machine update`](/docs/flyctl/machine-create/).
+
+## Usage
+
+Here's the usage of `fly machine run`:
+
+```cmd
+fly machine run <image> [command] [flags]
+```
+ 
+Here, `<image>` can point to a pre-built image, or to the current directory (`.`) to build from a Dockerfile.
+
+Many, but not all, Machine configuration options are available to the `fly machine run` command through flags. Flags are listed in the flyctl help and on the [`fly machine run` documentation page](/docs/flyctl/machine-run/).
+
+## Administration: set the Machine's app and org
+
+The default behavior of `fly machine run` is to create a new Fly App for the new Machine to belong to, unless it's given the name of an existing app in one of two ways:
+
+1. Like many other flyctl commands, `fly machine run` looks for a `fly.toml` app config file in the working directory, and if it finds one with an app name inside, it adds the new Machine to that app. It disregards the rest of the configuration in the file.
+2. If you pass it an app name with `--app <app-name>`, flyctl prefers that name over any name it gets from a `fly.toml`.
+
+If the app name doesn't belong to an existing app in one of your orgs, flyctl asks if you want to create it. 
+
+Even if you're not using `fly deploy` to configure and run any of your app's Machines, it may be worth creating a `fly.toml` file with just an app name in it, to save using the `--app` option repeatedly:
+
+```toml
+# a fly.toml just to provide an app name to commands 
+# run from the same directory
+
+app = <app-name>
+```
+ 
+Use `--org <org-name>` to specify which organization a newly created app should belong to. The `--org` flag is ignored when creating the new Machine in an existing app.
+
+
+## Get or build the Docker image
+
+All Fly Machines are made from Docker images. When you `fly launch` an app, a Fly Launch scanner may generate one for you based on your source code, or it may be a prebuilt image you point to, or it may be built from your hand-crafted Dockerfile and source code.
+
+We're working more directly with Machines here, so your choices with `fly machine run` are "point to an image" or "point to a Dockerfile."
```

Comment:
I found the introduction of "fly launch"ing an app a bit confusing. You can take or leave whatever from my suggestion below, which attempts to flip around the two scenarios so `fly m run` is first.
```suggestion
All Fly Machines are made from Docker images. When you work directly with Machines using `fly machine` commands, you can point to an prebuilt image, or build an image from your hand-crafted Dockerfile and source code.

When you `fly launch` an app, you can still point to image or point to Dockerfile, or a Fly Launch scanner may generate an image for you based on your source code.

```